x/ccv/consumer/keeper/genesis.go

```diff
@@ -118,35 +118,6 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) (genesis *consumertypes.GenesisSt
 			panic("provider client does not exist")
 		}
 
-		maturingPackets := []consumertypes.MaturingVSCPacket{}
```

Comment:
This logic is replaced by the `GetAllVSCPacketMaturityTimes` and `GetOutstandingDowntimes` calls below, correct?

Reply:
Yes